src/main/java/WayofTime/bloodmagic/ritual/crushing/ICrushingHandler.java

```diff
@@ -0,0 +1,11 @@
+package WayofTime.bloodmagic.ritual.crushing;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+public interface ICrushingHandler {
+    ItemStack getRecipeOutput(ItemStack input, World world, BlockPos pos);
```

Comment:
This should be marked as `@Nonnull` and any implementations should return `ItemStack.EMPTY` if they don't have an output.